src/transformers/modeling_utils.py

```diff
@@ -2742,7 +2742,10 @@ def save_pretrained(
         if module_map:
             filename_to_tensors = logging.tqdm(filename_to_tensors, desc="Saving checkpoint shards")
         for shard_file, tensors in filename_to_tensors:
-            shard = {tensor: state_dict[tensor] for tensor in tensors}
+            shard = {
+                tensor: state_dict[tensor] if state_dict[tensor].is_contiguous() else state_dict[tensor].contiguous()
```

Comment:
I don't think you need to check `is_contiguous` here - calling `contiguous` will just return the instance if it's already contiguous

Reply:
All right. More clever. Thanks!